test/compile/facet.test.ts

```diff
@@ -74,7 +74,7 @@ describe('FacetModel', () => {
     it('applies number format for fieldref of a quantitative field', () => {
       const model = parseFacetModelWithScale({
         facet: {
-          column: {field: 'a', type: 'quantitative', format: 'd'}
```

Comment:
Why dropping format here?

Reply:
I think it was invalid. I have to check again.

Reply:
Yep, we need `header: {format: 'f'}`

Reply:
I fixed it.